src/wp-includes/class-wp-theme-json.php

```diff
@@ -82,8 +82,8 @@ class WP_Theme_JSON {
 	 * - prevent_override => Disables override of default presets by theme presets.
 	 *                       The relationship between whether to override the defaults
 	 *                       and whether the defaults are enabled is inverse:
-	 *                         - If defaults are enabled  => theme presets should not be overriden
-	 *                         - If defaults are disabled => theme presets should be overriden
+	 *                         - If defaults are enabled  => theme presets should not be overridden
+	 *                         - If defaults are disabled => theme presets should be overridden
```

Comment:
This is not related to this PR, but while fixing a typo on another file I saw these and we might as well just fix them... It's just a typo fix in the docs so it has no impact in anything else.